index.html

```diff
@@ -312,6 +312,60 @@ <h1>SAVE Data (file0)</h1>
                 <label>Plot value</label>
                 <select title="Plot value" id="sav-plotvalue" class="w-full mt-1"></select>
             </fieldset>
+
+            <fieldset class="col-span-2">
+                <legend>FUN Values</legend>
+                <div class="grid-cols-2">
+                    <table>
+                        <tr>
+                            <th>Value</th>
+                            <th>Effect</th>
+                        </tr>
+                        <tr>
+                            <td>2-39</td>
+                            <td>Wrong number</td>
+                        </tr>
+                        <tr>
+                            <td>40-45</td>
+                            <td>Sans' call</td>
+                        </tr>
+                        <tr>
+                            <td>46-50</td>
+                            <td>Alpyhs' call</td>
+                        </tr>
+                        <tr>
+                            <td>56-57</td>
+                            <td>Nightmare mode</td>
+                        </tr>
+                        <tr>
+                            <td>61-63</td>
+                            <td>Gaster followers</td>
+                        </tr>
+                    </table>
+                    <table>
+                        <tr>
+                            <th>Value</th>
+                            <th>Effect</th>
+                        </tr>
+                        <tr>
+                            <td>65</td>
+                            <td>Sound test room</td>
+                        </tr>
+                        <tr>
+                            <td>66</td>
+                            <td>Fake hallway</td>
+                        </tr>
+                        <tr>
+                            <td>80-89</td>
+                            <td>Clam girl</td>
+                        </tr>
+                        <tr>
+                            <td>91-100</td>
```

Comment:
The `gml_Object_obj_mkid_goner_Create_0` script says:
```cpp
        if((global.flag[5] >= 90) && file_exists("undertale.ini")) {
            // ...
```
so I believe the right fun value is 91 and above, not 91-100.